keras/layers/dense_attention.py

```diff
@@ -242,6 +242,8 @@ class Attention(BaseDenseAttention):
       Defaults to `False`.
     dropout: Float between 0 and 1. Fraction of the units to drop for the
       attention scores. Defaults to 0.0.
+    score: One of {'dot', 'concat'}. 'dot' refers to dot multiplication
```

Comment:
Please rename argument to "score_mode" for consistency with other API conventions in Keras